app/views/delivery_updates/new.html.erb

```diff
@@ -0,0 +1,62 @@
+<% content_for :page_title,
+               "Add delivery update #{truncate(@study.title, length: 50)}" %>
+
+<% @delivery_update ||= DeliveryUpdate.new %>
+
+<div class="site-main site-main--impacts">
+    <div class="container">
+
+        <h4 class="add-outputs__title">Add delivery update for:</h4>
+        <div class="add-outputs__study">
+            <div class="study-header">
+                <%= render partial: "studies/header_primary" %>
+            </div>
+        </div>
+
+      <%= form_for @delivery_update,
+                  url: study_delivery_updates_path(@study),
+                  html: { class: "add-outputs__form action-sheet" } do |f| %>
+        <% if @invite_token.present? %>
+          <input type="hidden" name="token" value="<%= @invite_token %>">
+        <% end %>
+
+        <div class="action-sheet__section action-sheet__section--last">
+          <div class="row">
+            <p class="col-sm-12">
+              Please provide an update on how your study is progressing in
+              each of the delivery areas, detailing any delays with a brief
+              comment.
+            </p>
+          </div>
+          <div class="row">
+            <div class="col-sm-4">
+              <%= f.label :data_collection_status_id, "Data collection" %>
+              <%= f.collection_select :data_collection_status_id, DeliveryUpdateStatus.all, :id, :name %>
+            </div>
+            <div class="col-sm-4">
+              <%= f.label :data_analysis_status_id, "Data analysis" %>
+              <%= f.collection_select :data_analysis_status_id, DeliveryUpdateStatus.all, :id, :name %>
```

Comment:
@stevenday – How are statuses ordered in this `<select>` box? I'm seeing the following order, and it feels a bit funny:

1. Progressing fine
2. Not started
3. Minor…

Seems to me that "Not started" should go before "Progressing fine", because then the items have a sort of chronological order to them, rather than just being random.